Automattic/crowdsignal-ui
# Add loading state to the Submit Button

## Summary

The purpose of this PR is to add a loading state to the Submit Button when the user clicks it to send their responses.

Ref: c/vhhutFsG-tr

## Test Instructions

* Open or create a project with some blocks and a Submit Button
* Save it and go to the Preview
* Once you click the submit button you should see it getting into a loading state
